torch/_jit_internal.py

```diff
@@ -162,12 +164,11 @@ def parseExpr(expr, module):
 
 def createResolutionCallbackFromFrame(frames_up: int = 0):
     """
-    Creates a function which, given a string variable name,
-    returns the value of the variable in the scope of the caller of
-    the function which called createResolutionCallbackFromFrame (by default).
+    Return the value of the variable in the scope of the caller of the calling function given a string variable name.
```

Comment:
```suggestion
    Create a function which, given a string variable name, returns the value of the variable in the scope of the caller of the function which called createResolutionCallbackFromFrame (by default).
```

Reply:
What do you think of the revision below? I tried implementing your suggestion but ran into the 120-character line limit.
```
    Create a function which returns a string variable's value, given its name, from the scope of the caller's caller.

    Returns the value of the variable in the scope of the caller of the function which called
    createResolutionCallbackFromFrame (by default).
    
	...
```